src/OpenColorIO/MathUtils.h

```diff
@@ -30,101 +30,89 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef INCLUDED_OCIO_MATHUTILS_H
 #define INCLUDED_OCIO_MATHUTILS_H
 
-#include <cmath>
-#include <vector>
+#if defined(_WIN32) || defined(_WIN64) || defined(_WINDOWS) || defined(_MSC_VER)
+#ifndef WINDOWS
+#define WINDOWS
+#endif
+#endif
+
```

Comment:
Is this to avoid min/max pollution or something?  would defining  NOMINMAX be less ugly if so?

Reply:
That's dead code I forgot to remove in my various attempts to fix the Clang 9 warnings.

Reply:
Fixed.